services/treeware/treeware-trees.tester.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
+const { isMetric } = require('../test-validators')
+
+t.create('request for existing package')
+  .timeout(10000)
```

Comment:
Is this required, or is it just copied from another test? If our starting assumption is that this will regularly take ~10 seconds to give us a response, I'm a bit concerned that these badges are mostly going to fail to render. That said, all the example calls I've made to this API returned a response quickly.

Reply:
Ah, I copied it from another badge. Just assumed it was required, but I'll remove it. :+1: